tests/inputs/q-search-filter/error_filter_set2.py

```diff
@@ -0,0 +1,16 @@
+from nautobot.apps.filters import NautobotFilterSet, SearchFilter
+
+
+class CustomSearchFilter(SearchFilter):
+    """Custom search filter."""
+
+    filter_predicates = {
+        "name": ["name__icontains"],
+        "description": ["description__icontains"],
+    }
+
+
+class MyAddressObjectFilterSet(NautobotFilterSet):
```

Comment:
Is this supposed to fail due to an incorrect Q filter type? I would think this should pass because the q filter is a subclass of `SearchFilter`

Reply:
We can't check for subclass though in AST. This is meant to fail, but is a separate code so that we can optionally disable it.